styles/main.css

```diff
@@ -30,28 +30,30 @@ nav a {
     padding-bottom:0.5em;
     margin-left:1em;
     margin-right:1em;
+    border-top-width: 4px;
     border-style:solid;
     border-color:transparent;
-
+    display: block;
 }
 
 nav ul {
     list-style: none;
+    float:right;
 }
 
 nav li{
     float:left;
 }
 
 header nav {
-    float:right;
     border-bottom: 1px dotted ;
     border-bottom-color: rgba(0, 0, 0, .3);
     margin-right:2em;
     margin-left:2em;
     margin-top:-4px;
     height:auto;
     border-width:auto;
+    overflow: auto;

```

Comment:
This is the declaration that makes `nav` take on the height of its floated contents